Cargo.toml

```diff
@@ -11,7 +11,7 @@ categories = ["web-programming::http-client"]
 
 [dependencies]
 hyper = "0.10.2"
-hyper-native-tls = "0.2"
+hyper-native-tls = { git = "https://github.com/sfackler/hyper-native-tls" }
```

Comment:
Just making a note here for myself to not merge until we can remove the git dependency.